### DISCUSSION TITLE
为什么不提供man xmake来查看文档呢?

### DISCUSSION THREAD

**Discussion by etcix** [xmake-io/xmake#5038] [Ideas]
我想使用man xmake,而不是去联网看官方文档

**Comment by waruqi** [comment]
没有为什么，一个人的时间精力是有限的，现在一份文档都维护不过来。

如果想要类似 man 的离线文档，可以先提 pr 过来帮我们一起完成迁移，翻译全部的 API 手册文档。

https://github.com/xmake-io/xmake-gendoc

我们正在重构文档框架，先要补全所有的 API 文档。然后我们才会考虑提供一个 `xmake man` 的插件命令，来快速离线查看特定 API 的文档描述。

这个工作量是巨大的，靠我一个人维护，根本没这么时间，我还要处理各种 issues 和新特性开发。

所以文档这块，我目前的空闲时间，只能做到发版前，更新下当前已有的在线文档，补全新的特性和改动。
